### ISSUE TITLE
directory searx

### ISSUE THREAD

**Issue by aleferov** [searx/searx#1150]
Hello! How to remove from the address / searx? That was domain.com, not domain.com/searx.

**Comment by thiswillbeyourgithub** [comment]
I'm not a contributor but I don't think your question is clear. If I were you I'd rephrase this to reduce their workload :)

**Comment by aleferov** [comment]
English is not my native language(((

I installed in a subfolder Searx (domain.com/searx). And now I want to Searx worked at domain.com .

**Comment by ghost** [comment]
This depends on your web server. Take a look at https://asciimoo.github.io/searx/dev/install/installation.html#web-server

**Comment by aleferov** [comment]
Thank very much!
